### PR TITLE
Docs 9771 audit log cleanup

### PR DESCRIPTION
**Pull request by sarathecoubian**

### CODE REVIEW

File: modules/ROOT/pages/audit-logging.adoc [hunk 1]
```diff
@@ -1362,510 +1383,538 @@ properties: name, isPublic
...
-|subaction : none +
-properties : application metadata
-|Start Application
+|Subaction: None +
+Properties: Application metadata
+|Start application
 |Application
 |AppID
-|n/a
+|N/A
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/audit-logging.adoc:1599]: Is the "Payload" cell empty on purpose for this one entry?

**Review by Mulesoft-Mysti — APPROVED** [review]
The last cell for Runtime Manager > Create Application don't show up in the View File version of this doc. Not sure if there's a real issue or not.

File: modules/ROOT/pages/audit-logging.adoc (hunk 1 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/audit-logging.adoc:1599]: I'd be shocked if it were. I will reach out to the appropriate party to ask. Good catch!!

File: modules/ROOT/pages/audit-logging.adoc [hunk 2]
```diff
@@ -1362,510 +1383,538 @@ properties: name, isPublic
...
 |Start
 |
```
Comment (sarathecoubian) [inline modules/ROOT/pages/audit-logging.adoc:1601]:
```suggestion
|Subaction: None +
Properties: Application metadata
```

Comment (sarathecoubian) [reply modules/ROOT/pages/audit-logging.adoc:1601]: (need to verify)

**Pull request merged**